packages/mui-system/src/cssVars/createCssVarsProvider.js

```diff
@@ -41,6 +45,21 @@ export default function createCssVarsProvider(options) {
     return value;
   };
 
+  const disableCSSTransition = () => {
```

Comment:
```suggestion
  // credit: https://github.com/pacocoursey/next-themes/blob/b5c2bad50de2d61ad7b52a9c5cdc801a78507d7a/index.tsx#L313
  const disableCSSTransition = () => {
```

I think it is nice to add credit for the author.